docs/translating-dev.md

```diff
@@ -6,10 +6,16 @@
 - Be able to understand English
 - Be able to understand the language you want to translate riot-web into
 
+## Translating strings vs. marking strings for translation
+
+Translating strings are done with the `_t()` function found in matrix-react-sdk/lib/languageHandler.js. It is recommended to call this function wherever you introduce a string constant which should be translated. However, translating can not be performed until after the translation system has been initialized. Thus, sometimes translation must be performed at a different location in the source code than where the string is introduced. This breaks some tooling and makes it difficult to find translatable strings. Therefore, there is the alternative `_td()` function which is used to mark strings for translation, without actually performig the translation (which must still be performed separately, and after the translation system has been initialized).
```

Comment:
typo: "performig"

Reply:
I think it would be useful to provide a small code-snippet example of a string being defined as a module constant, and then later translating at the point of rendering.